### DISCUSSION TITLE
Hire Me (Feb 2024)

### DISCUSSION THREAD

**Discussion by manovotny** [vercel/next.js#61897] [Show and tell]
# What is this?

This discussion is for Next.js developers looking for work.

## Candidates

Share your information if you are looking for work.

Please use this format:

> Location:
> Remote:
> Willing to relocate:
> Technologies:
> Resume/CV:
> Website:
> Email:

Good luck with interviewing!

---

If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍

If you're a company looking to hire, please use #61896 instead.

We've had yet another [success story](https://twitter.com/nutlope/status/1610667757565083648) of a developer being hired!

**Comment by jensen** [comment]
Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript and Mobile Applications  (Check GitHub)
Resume/CV: https://www.linkedin.com/in/karljensendev/
Website: https://github.com/jensen

Previously: EA Sports
Recently shipped: https://alicent.ai/, https://rlife.app

In 2021/2022 I built over **20** web applications which can be found on my GitHub profile.

**I am only looking for contract work at this time.**

**Comment by ryanfurrer** [comment]
**Location**: Islip, NY, USA
**Remote**: Yes, willing to go into office if required
**Willing to relocate**: Yes, if it is near Spring, TX, USA 
**Technologies**: Front-End Web Developer with a focus on design: HTML, CSS, SASS/SCSS, Tailwind CSS, JavaScript, Astro, React/NextJS, Git, GitHub
**Resume/CV**: [Resume](https://ryanfurrer.com/_astro/ryan-furrer-resume_webdev.TbezTf9w.pdf) | [CV (LinkedIn)](https://www.linkedin.com/in/ryanfurrer/)
**Website**: [ryanfurrer.com](https://ryanfurrer.com)
**Email**: ryandotfurrer@gmail.com

**Comment by Khaan25** [comment]
Location: Rawalpindi, Pakistan
Remote: Yes
Willing to relocate: No
Technologies: listed here - https://zzia.vercel.app/about
Resume/CV: https://zzia.vercel.app/ziacodes.pdf || https://www.linkedin.com/in/ziawebdev
Website: https://zzia.vercel.app
Email: zia@vennsol.pk || ziaurzai@gmail.com
![og](https://github.com/vercel/next.js/assets/72739794/ad3f44bd-9100-4486-a1f5-4c38d8131abd)

**Comment by Eagle9461** [comment]
Location: Krakow, Poland
Remote: Yes
Willing to relocate: No
Technologies: 
Resume/CV: https://drive.google.com/file/d/1HcCZWy6Ld4-sa106Bye5gjEsxYhQf5KE/view?usp=sharing
Website:
Email: eagle9995.an@gmail.com

**Comment by albertkimdev** [comment]
Location: Toronto, Canada
Remote: Yes
Willing to relocate: To NYC
Technologies: React, Next, GSAP, Framer Motion, Tailwind
Resume/CV: On linkedin which is on my website
Website: https://albertkim.co
Email: albert@albertkim.co

**Comment by superdev87** [comment]
Location: Lippstadt, Germany
Remote: Yes
Willing to Relocate: No
Technologies: React, Next.js, SASS/SCSS, TailwindCSS, TypeScript
Linkedin: https://www.linkedin.com/in/alexander-heck-b649a62a3
Email: alexanderheck429@gmail.com

**Comment by jarrensj** [comment]
Location: Irvine, CA
Remote: Yes
Willing to relocate: LA, Northern California, New York
Technologies: Next, React, TypeScript, JavaScript, Node, AWS, HTML, CSS, Tailwind
Resume/CV: https://www.linkedin.com/in/jarrensanjose
Website: http://jarrensj.com
Email: sanjose.jarren@gmail.com

**Comment by hhagely** [comment]
Location: St. Louis, MO
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, TypeScript, React, NextJs, Node, GraphQL, HTML, CSS, 
LinkedIn: https://www.linkedin.com/in/herb-hagely-b22228bb
Email: herb.hagely@gmail.com

I recently was on a team that implemented the WIC program for a major US grocery chain. Assisting in leading the team in tasks, shaping the direction architecturally and helping other engineers grow.

**Comment by emapeire** [comment]
Location: Based in Rosario, Argentina 🇦🇷
Remote: Yes
Willing to relocate: Yes
Technologies: HTML, CSS, Tailwind-CSS, Astro, Next.js, React, Svelte, SQL, PostgreSQL, MySQL, Shadcn/ui.
Resume/CV: [CV | Emanuel Peire](https://cv.emapeire.xyz)
Website: [Portfolio | Emanuel Peire](https://emapeire.xyz)
Email: [me@emapeire.xyz](mailto:me@emapeire.xyz)

**Comment by ibarczewski** [comment]
Location: Seattle, WA
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, TypeScript, React, NextJs, Vue, Angular, Node, MySQL, SQL Server, HTML, CSS, C#, Java, Kotlin
LinkedIn: https://www.linkedin.com/in/ibarczewski
Email: [ibarczewski@gmail.com](mailto:ibarczewski@gmail.com)
[Resume Here!](https://drive.google.com/file/d/1z8etoHb94Z0lMpaS-zm5G-yXxeAJE2Gi/view?usp=sharing)

**Comment by T1LT** [comment]
#### Hi! I’m Nishant, a Software Engineer with a drive to elevate people’s web experience.

Location: Cupertino, CA
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js, React, Redux, TypeScript, JavaScript, Tailwind CSS, HTML, CSS, Drizzle, Prisma, Node.js, Express, Ruby on Rails, PostgreSQL, MongoDB, Shadcn UI, Radix UI, MUI
Resume: [link to resume](https://nishantracherla.com/resume.pdf)
Website: [nishantracherla.com](https://nishantracherla.com)
Email: nishant.racherla@gmail.com

**Comment by joelcipriano** [comment]
Location: Recife, Brazil.
Remote: Yes.
Willing to relocate: Not at the moment.
Technologies: Angular, React, Next, Turborepo, Postgres, Firebase.
Resume/CV: https://linkedin.com/in/joelcipriano
Website: https://joelcipriano.com.br/
Email: joelcipriano@gmail.com

**Comment by newaritra** [comment]
Location: India
Remote: Yes
Willing to relocate: No
Technologies: Next.js, CSS, Node, Express, SQL, Tailwind
Resume/CV: [Resume](https://drive.google.com/file/d/1rQ_fMKav-IZH1Uu59aCo72BMH5dLGrKP/view?usp=drivesdk) | [LinkedIn](https://www.linkedin.com/in/aritra-roy-webdev?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app)
Email: aritraroy14.ar@gmail.com

**Comment by apophis51** [comment]
### Location: Largo, Florida, USA
### Remote: YES
### Willing to relocate: YES
### Technologies:

- Languages/Scripting: JavaScript, Python, Solidity, Bash, SQL
- Front-end Frameworks/Libraries: React, Redux, TailwindCss, Three.js, Framer Motion, MaterialUI, DaisyUi,
- Shopify .Liquid
- Back-end/Frameworks: Node, Next.js, Flask, Express.js, Astro.js
- OS and Source Control: Windows, Linux, Git, GitHub
- Deployment: AWS (EC2), Google (compute) Railway, Vercel, Netlify, Docker
- Markup: HTML5, CSS, Markdown
- Databases: PostgreSql
- Other Libraries: Axios, Selenium, Sequelize, Prisma
- Dabbled with: Mokoto, React Native, ScikitLearn, Pandas, FireBase, MongoDb
- Architectural Approaches: JamStack, SPA, MPA, MVC
- 
### Website: https://malcmind.com
### Email: admin@malcmind.com

**Comment by ryanhenning** [comment]
Location: St. Louis, MO
Remote: Yes
Willing to relocate: No
Technologies: React, React Native, Next.js, Node, Javascript, Typescript, C#, SQL, MongoDB, Docker, Kubernetes
LinkedIn: https://linkedin.com/in/ryan-henning-developer
Resume: [View Here](https://drive.google.com/file/d/1doB35VQJMVHl-vyo5F-zzMtGF9BIBYV4/view)
Email: [ryan.henning@gmail.com](mailto:ryan.henning@gmail.com)

**Comment by nemanjam** [comment]
Location: Serbia, GMT +1
Remote: Yes
Willing to relocate: Maybe
Technologies: React, Next.js, JavaScript/TypeScript, Node.js, TailwindCSS, Jest, Cypress, Prisma, PostrgreSQL, MongoDB, Docker, C#/.NET
Resume/CV: Available on request, please send me an email
GitHub: https://github.com/nemanjam
LinkedIn: https://www.linkedin.com/in/nemanja-mitic
Email: nemanja.mitic.elfak [at] hotmail.com


I am a full stack developer with a Master's degree in Computer Science and 4 years of professional experience with JavaScript and C#/.NET. In the last few years my main focus has been the React/Next.js ecosystem, followed by Node.js. I am a JavaScript professional and a Python beginner interested to get more exposure to it. I have also worked with ASP.NET in the past which is not my primary focus at the moment. I have a basic understanding of DevOps tools.

A quick and easy way to assess my current skill level, the quality of my code and documentation, and my preferred tech stack (Next.js, TypeScript, React Query, Prisma, Postgres, TailwindCSS, Docker, Github Actions) is to check out one of my latest projects: https://github.com/nemanjam/nextjs-prisma-boilerplate .

I have experience working remotely for startups within globally distributed multicultural teams.

I am looking for a full-time remote position. We can discuss relocation after some time spent in the company. You can check out my other projects at https://github.com/nemanjam . I have detailed resume available, you can tell me about your company and project by email: nemanja.mitic.elfak [at] hotmail.com or by LinkedIn: https://www.linkedin.com/in/nemanja-mitic .

**Comment by dlarroder** [comment]
Location: Philippines
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, Storybook.js, JavaScript/TypeScript, Node.js, TailwindCSS, Jest, Firebase, Redux Toolkit
Website: https://www.dalelarroder.com/
Email: dlarroder@gmail.com

**Comment by chloegng** [comment]
Location: Bay Area, California
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, Node.js, JavaScript, TypeScript, HTML, CSS, Tailwind-CSS, Cypress, SQL, PostgreSQL, MySQL, Solidity, Python, Vercel, AWS, Liquid
Resume/CV: [https://www.chloegng.com/Chloe-Ng-Resume.pdf](https://www.chloegng.com/Chloe-Ng-Resume.pdf)
Website: [https://www.chloegng.com/](https://www.chloegng.com/)
Email: [chloegng@gmail.com](mailto:chloegng@gmail.com)

**Comment by karan0805** [comment]
Location: Pune, India
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, Node.js, JavaScript, TypeScript, HTML, CSS, Tailwind-CSS, SQL, PostgreSQL, MySQL, Python, Vercel, AWS, Liquid
Resume/CV: https://karan0805.vercel.app/resume
Website: https://karan0805.vercel.app/
Email: [karangandhi.navsari@gmail.com](mailto:karangandhi.navsari@gmail.com)

**Comment by suefeng** [comment]
Location: Chicago, Illinois, USA
Remote: Yes
Willing to relocate: No
Technologies: HTML5, CSS3, JavaScript, SQL, PHP, Ruby, React, Next.js, Rails, MySQL, Tailwind CSS, Bootstrap CSS
Resume/CV: https://suefeng.com/resume
Website: https://suefeng.com
Email: suefengtan@gmail.com

**Comment by FLCN-16** [comment]
Location: India
Remote: yes
Willing to relocate: yes
Technologies: Next.JS, React, Node.JS, TypeScript, GraphQL, Tailwind, Python, MySQL, MongoDB, Vercel, AWS, GCP
Resume/CV: https://www.linkedin.com/in/flcn16/
Website: https://github.com/FLCN-16
Email: work@thefalcon.dev

**Comment by VenkateshKoka** [comment]
Hi, I am Venkatesh Koka. Having worked in the United States(work experience 3.5 years) in companies like **Amazon and Drafted(startup)**, I had to return to India due to Work Visa issues.

Location: India
Remote: yes
Willing to relocate: yes
Technologies: React, Next.JS, Node.JS, GraphQL, Tailwind, SCSS, Python, MySQL, MongoDB, Vercel, AWS, MongoDB, PostgreSQL, Machine Learning
Resume/CV: https://www.linkedin.com/in/venkateshkoka45
Website: https://www.venkateshkoka.com and https://github.com/VenkateshKoka/
Email: venkateshkoka45@gmail.com
